lib/ProductOpener/Ingredients.pm

```diff
@@ -144,31 +144,46 @@ my $separators = qr/($stops\s|$commas|$separators_except_comma)/i;
 
 # do not add sub ( ) in the regexps below as it would change which parts gets matched in $1, $2 etc. in other regexps that use those regexps
 # put the longest strings first, so that we can match "possible traces" before "traces"
-my %traces_regexps = (
+my %may_contain_regexps = (
 
 	en => "possible traces|traces|may contain",
 	de => "Kann Spuren|Spuren",
 	es => "puede contener|trazas|traza",
 	fi => "saattaa sisältää pieniä määriä muita|saattaa sisältää pieniä määriä|saattaa sisältää pienehköjä määriä muita|saattaa sisältää pienehköjä määriä|saattaa sisältää",
 	fr => "peut contenir|qui utilise|utilisant|qui utilise aussi|qui manipule|manipulisant|qui manipule aussi|traces possibles|traces d'allergènes potentielles|trace possible|traces potentielles|trace potentielle|traces éventuelles|traces eventuelles|trace éventuelle|trace eventuelle|traces|trace",
 	it => "può contenere|puo contenere|che utilizza anche|possibili tracce|eventuali tracce|possibile traccia|eventuale traccia|tracce|traccia",
+);
+
+my %contains_regexps = (
 
+	en => "contains",
+	de => "Kann",
```

Comment:
"Kann" is literally "can" while everything else is "enthalten".

Reply:
Thanks @hangy . I'll remove my attempt at handling German from this pull request, we can add proper support later.